src/Core/SharedKernels/Enumerator/Enumerator/ViewModels/InterviewDetails/Questions/CategoricalComboboxAutocompleteViewModel.cs

```diff
@@ -25,6 +25,8 @@ public class CategoricalComboboxAutocompleteViewModel : MvxNotifyPropertyChanged
         private readonly bool displaySelectedValue;
         private readonly ThrottlingViewModel throttlingModel;
         private readonly IMvxMainThreadAsyncDispatcher mvxMainThreadDispatcher;
+        
+        private readonly int LoadingOptionValue = int.MinValue;

```

Comment:
maybe to have nullable value and null if not loading is safer? if someone selects int.MinValue as option value?

Reply:
it old code, this field is removed